Mlem/Views/Shared/Posts/Feed Post.swift

```diff
@@ -149,16 +149,30 @@ struct FeedPost: View {
             with: ConcreteEditorModel(post: postModel, operation: PostOperation.replyToPost)
         )
     }
+    
+    /// Render read pinned posts in less "in-your-face" way.
+    private var renderPinnedAsCompact: Bool {
+        /// Only render pinned posts in compact size in Community feed, ignore this behaviour in other feed types (e.g. Aggregate). [2024.01]
+        guard let community, postTracker?.feedType == .community(community) else {
```

Comment:
Something's funky with this logic--it's not consistently working when I test locally. This should do the trick:
```suggestion
        guard case .community = postTracker?.feedType else {
```